static/js/services/delete-docs.js

```diff
@@ -78,8 +75,15 @@ var _ = require('underscore');
             return checkForDuplicates(docs);
           })
           .then(function() {
-            return DB().bulkDocs(docs);
+            return docs.map(function(doc) {
```

Comment:
I think this has changed the code so now it also deletes the parent. That's probably bad, right?